migrations/m180307_205154_media_href.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use yii\db\Migration;
+
+class m180307_205154_media_href extends Migration
```

Comment:
This has nothing todo with sortable galleries, please open another PR for this feature and describe why you need/want a href field.